<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 228, 228);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QToolButton" name="loginBtn">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 30px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }</string>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::SystemLogOut"/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="orderBtn">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 30px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }</string>
         </property>
         <property name="text">
          <string>行程</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::DocumentNew"/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QToolButton" name="userInfoBtn">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 30px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
              background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }</string>
         </property>
         <property name="text">
          <string>我的</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="mainBtn">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton{
              border-top: 3px outset transparent;
              border-bottom: 7px outset transparent;
              border-right: 30px outset transparent;
              border-left: 20px outset transparent;

              background-color: rgb(228, 228, 228);
        }

        /* 鼠标悬停 */
        QToolButton:hover{
              background-color: rgb(205, 205, 205);
        }

        /* 点击和按下 */
        QToolButton:pressed,QToolButton:checked{
    			background-color: rgb(246, 246, 246);
        }

        QPushButton:default {
          border-color: navy; /* make the default button prominent */
        }</string>
         </property>
         <property name="text">
          <string>主页</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::GoHome"/>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
